### DISCUSSION TITLE
Vercel Analytics Manual Pageviews with Segment

### DISCUSSION THREAD

**Discussion by daneFFW** [vercel/community#3447] [Feature Question] [answered]
### Summary

Hello, 
I am attempting to trigger pageviews manually. The goal is to use segment to trigger the pageview calls instead of them being triggered automatically from the script. 
Your analytics documentation found here (https://vercel.com/docs/concepts/analytics/custom-events) specifies the va.track() 
```
va('event', {
  name: 'Signup',
  data: {
    location: 'footer',
  },
});
```

but there is no documentation calling the view/pageview or what needs to be included in a manual call.

Please advise.
Thanks

### Steps to Reproduce

Nothing to reproduce, just need advice on the above.

**Comment by amyegan** [comment, marked as answer]
Hey @daneFFW. You can pass a string representing the event name as the first argument. An object with string, number, or boolean key-value pairs can be passed as the second argument to include custom data. It's up to you to decide which events should trigger custom tracking and what data should be included.

Keep in mind that custom events are only available on Pro and Enterprise plans and you'll need to have have version 1.0.0 or later of `@vercel/analytics` installed.

**Comment by daneFFW** [comment]
Hi @amyegan Thanks for the response but I am trying to send pageviews/views manually not custom events at this time. It looks like there is the  view endpoint that receives a json object with 4 key value pairs. See below from my hobby account. 
{"o":"https://verceltest-gray-ten.vercel.app/","sv":"0.1.2","ts":1690831601359,"r":"https://vercel.com/"} 
It looks like there are 4 key value pairs, 
o: current url
sv: version?
ts: timestamp
r: referrrer
it also looks like there is another parameter that is not required 
 sdkn:
 Is this correct?
